vue-client/src/utils/data.js

```diff
@@ -164,6 +164,15 @@ export async function fetchMissingLinkedToQuoted(obj, store) {
     .catch(e => console.log(e));
 }
 
+export function moveWorkToInstance(data) {
+  const oldWork = data.work;
+  if (oldWork !== undefined) {
+    delete oldWork['@id']; // Get rid of <record-id>#work
```

Comment:
Makes sense (and is "correct"); I just wonder if we want some sort of cue to show that something "happened" to the work entity (that is was embedded). We've only done this once though, so building view logic for that is arguably overkill.

Reply:
Yes, I'm not 100% sure how the diff will look for the version where embedding took place. Probably it will look like all items in instanceOf were newly added. We can take another iteration for that specific case if it's unclear/misleading.